crates/plume/src/lib.nr

```diff
@@ -109,7 +110,7 @@ fn sha256_12_coordinates(
         }
     }
 
-    sha256(res)
+    hash([to_field(res)], res.len())
```

Comment:
This looks like we are missing on preimage data when doing conversion to field; `res`  being 198 bytes of information is narrowed down to the prime order of underlying field of appr. 8 bytes for bb prior to hashing. Additionally, here is passed only one field element in array, but `res.len()` will give 198.

Reply:
I see, so in this case do you suggest I break it to limbs and pass it as an array maybe?